sdlib/d/gc/arena.d

```diff
@@ -165,6 +165,7 @@ public:
 package:
 	Extent* allocSlab(shared(ExtentMap)* emap, ubyte sizeClass) shared {
 		auto ec = ExtentClass.slab(sizeClass);
+		import d.gc.slab;
```

Comment:
It's a bit inelegant to have the import int he middle here and referring to something in the middle of an expression. It's probably worth extracting the binInfos access on its own line.